sphinx-docs/Plugin-library.md

```diff
@@ -168,6 +174,8 @@ The Atomic plugin imports all Red Canary Atomic tests from their open-source Git
 The GameBoard plugin allows you to monitor both red-and-blue team operations. The game tracks points for both sides
 and determines which one is "winning". 
 
+To begin a gameboard exercise, first log in as blue user and begin an operation. Then log in as red user and begin another operation. Open up the gameboard plugin from the GUI and select these new respective red and blue operations to monitor points for each operation. 
```

Comment:
Auto-collect operation will be executed automatically if you start a blue agent. Though you can login as blue and start an op simply running a blue agent is sufficient